### PR TITLE
[FE] fixed the head.ejs

### PR DESCRIPTION
**Pull request by AliceHuang1027**
commit again and it shall fix the issue of include head.ejs in all the other files

### CODE REVIEW

File: views/partials/head.ejs [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <!--views/partials/head.ejs-->
 <meta charset='UTF-8'>
-<title>Welcome to learndb.dev </title>
-
+<title></title>
```
Comment (songz) [inline views/partials/head.ejs:3]: couldy ou delete this tag if you don't need it?

Comment (AliceHuang1027) [reply views/partials/head.ejs:3]: okay

**Comment by AliceHuang1027** [comment]
Here are how the three pages look like (signin, signup, setDBpassword) :

Screen Shot 2020-07-13 at 4 29 38 PM
Screen Shot 2020-07-13 at 4 29 38 PM
Screen Shot 2020-07-13 at 4 30 45 PM

File: views/signup.ejs [hunk 2]
```diff
@@ -1,26 +1,4 @@
...
-        width: 400px;
-        height: 40px;
-        margin: 10px;
-    }
-    .body {
-        margin: 10px;
-    }
-</style>
-
+<%- include('./partials/head') %>
```
Comment (AliceHuang1027) [inline views/signup.ejs:1]: How to solve "EJS Unexpected token / - Using Partials" issue: https://cmsdk.com/node-js/ejs-unexpected-token---using-partials.html 👻

**Pull request merged**